From 268ad088003abc05e795c8133f036c4847b67123 Mon Sep 17 00:00:00 2001
From: Avishek Sen <x0rzavi@gmail.com>
Date: Wed, 6 Sep 2023 09:55:42 +0530
Subject: [PATCH 09/18] performance kernel cmdline

Signed-off-by: Avishek Sen <x0rzavi@gmail.com>
---
 configs/releng/grub/grub.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configs/releng/grub/grub.cfg b/configs/releng/grub/grub.cfg
index fcf1bd3..fa80b24 100644
--- a/configs/releng/grub/grub.cfg
+++ b/configs/releng/grub/grub.cfg
@@ -45,7 +45,7 @@ play 600 988 1 1319 4
 menuentry "Arch Linux install medium (x86_64, UEFI)" --class arch --class gnu-linux --class gnu --class os --id 'archlinux' {
     set gfxpayload=keep
     # linux /%INSTALL_DIR%/boot/x86_64/vmlinuz-linux archisobasedir=%INSTALL_DIR% archisodevice=UUID=${ARCHISO_UUID}
-    linux /%INSTALL_DIR%/boot/x86_64/vmlinuz-linux-cachyos archisobasedir=%INSTALL_DIR% archisodevice=UUID=${ARCHISO_UUID} cow_spacesize=6G
+    linux /%INSTALL_DIR%/boot/x86_64/vmlinuz-linux-cachyos archisobasedir=%INSTALL_DIR% archisodevice=UUID=${ARCHISO_UUID} cow_spacesize=6G module_blacklist=sp5100_tco,iTCO_wdt nmi_watchdog=0 nowatchdog mitigations=off
     # initrd /%INSTALL_DIR%/boot/intel-ucode.img /%INSTALL_DIR%/boot/amd-ucode.img /%INSTALL_DIR%/boot/x86_64/initramfs-linux.img
     initrd /%INSTALL_DIR%/boot/intel-ucode.img /%INSTALL_DIR%/boot/amd-ucode.img /%INSTALL_DIR%/boot/x86_64/initramfs-linux-cachyos.img
 }
@@ -53,7 +53,7 @@ menuentry "Arch Linux install medium (x86_64, UEFI)" --class arch --class gnu-li
 menuentry "Arch Linux install medium with speakup screen reader (x86_64, UEFI)" --hotkey s --class arch --class gnu-linux --class gnu --class os --id 'archlinux-accessibility' {
     set gfxpayload=keep
     # linux /%INSTALL_DIR%/boot/x86_64/vmlinuz-linux archisobasedir=%INSTALL_DIR% archisodevice=UUID=${ARCHISO_UUID} accessibility=on
-    linux /%INSTALL_DIR%/boot/x86_64/vmlinuz-linux-cachyos archisobasedir=%INSTALL_DIR% archisodevice=UUID=${ARCHISO_UUID} accessibility=on cow_spacesize=6G
+    linux /%INSTALL_DIR%/boot/x86_64/vmlinuz-linux-cachyos archisobasedir=%INSTALL_DIR% archisodevice=UUID=${ARCHISO_UUID} accessibility=on cow_spacesize=6G module_blacklist=sp5100_tco,iTCO_wdt nmi_watchdog=0 nowatchdog mitigations=off
     # initrd /%INSTALL_DIR%/boot/intel-ucode.img /%INSTALL_DIR%/boot/amd-ucode.img /%INSTALL_DIR%/boot/x86_64/initramfs-linux.img
     initrd /%INSTALL_DIR%/boot/intel-ucode.img /%INSTALL_DIR%/boot/amd-ucode.img /%INSTALL_DIR%/boot/x86_64/initramfs-linux-cachyos.img
 }
-- 
2.42.0

