From ba0d8eb10d644593be5fd64576e99230cc9cc4e8 Mon Sep 17 00:00:00 2001
From: Avishek Sen <x0rzavi@gmail.com>
Date: Sun, 17 Sep 2023 10:40:10 +0530
Subject: [PATCH] Required packages

Signed-off-by: Avishek Sen <x0rzavi@gmail.com>
---
 configs/releng/packages.x86_64 | 46 ++++++++++++++++++++++++++++++++--
 1 file changed, 44 insertions(+), 2 deletions(-)

diff --git a/configs/releng/packages.x86_64 b/configs/releng/packages.x86_64
index 519b950..dc77a98 100644
--- a/configs/releng/packages.x86_64
+++ b/configs/releng/packages.x86_64
@@ -127,8 +127,50 @@ xfsprogs
 xl2tpd
 zsh
 
-# needs chaotic-AUR GPG keys on build machine
+# needs chaotic-AUR repo GPG keys on build machine
 linux-cachyos
 linux-cachyos-headers
+chaotic-keyring
+chaotic-mirrorlist
+ryzenadj-git
+zenpower3-dkms
+zenmonitor3-git
 
-archzfs-dkms-git
\ No newline at end of file
+# needs archzfs repo GPG keys on build machine
+# archzfs-linux-git
+
+# core
+archzfs-dkms-git
+dash
+hyprland
+paru
+nbfc-linux
+
+# fonts
+noto-fonts
+noto-fonts-cjk
+noto-fonts-emoji
+
+# drivers
+mesa
+libva-mesa-driver
+vulkan-radeon
+xf86-video-amdgpu
+
+# sound
+pipewire
+pipewire-alsa
+pipewire-audio
+pipewire-pulse
+pipewire-jack
+wireplumber
+
+# utility
+aria2
+btop
+byobu
+fastfetch
+firefox
+neovim
+p7zip
+kitty
-- 
2.42.0

